src/debugAdapter/mplabxDebug.ts

```diff
@@ -199,7 +199,7 @@ export class MplabxDebugSession extends LoggingDebugSession {
 	protected async launchRequest(response: DebugProtocol.LaunchResponse, args: ILaunchRequestArguments) {
 
 		// start the program in the runtime
-		this._runtime.startDebugger(args.program, args.configuration, !!args.stopOnEntry).then(r => {
+		this._runtime.startDebugger(args.program, args.configuration, !args.noDebug, !!args.stopOnEntry).then(r => {
```

Comment:
package.json defins a `debug` Debug argument, but here a `noDebug` argument is called. please correct this line to use `debug` instead of `noDebug`.

Reply:
Yeah for some reason I didn't realize that the type for `ILaunchArguments` was owned by the extension, so I used `noDebug` from the inherited arguments, but I've fixed it now. Good spot!